### PR TITLE
[kadena-cli] improve password texts

### PR DESCRIPTION
**Pull request by barthuijgen**

### CODE REVIEW

File: packages/tools/kadena-cli/src/wallets/commands/walletsWalletGenerate.ts [hunk 1]
```diff
@@ -97,7 +97,10 @@ export const createGenerateWalletCommand: (
   'Add a new local wallet',
   [
     globalOptions.walletName({ isOptional: false }),
-    globalOptions.passwordFileRepeat({ isOptional: false }),
+    securityOptions.createPasswordOption({
+      message: 'Enter the new wallet password',
+      confirmPasswordMessage: 'Enter the password again',
```
Comment (nillo) [inline packages/tools/kadena-cli/src/wallets/commands/walletsWalletGenerate.ts:102]: re-enter the password

**Review by nillo — APPROVED** [review]

File: packages/tools/kadena-cli/src/wallets/commands/walletsImportWallet.ts [hunk 2]
```diff
@@ -70,7 +70,10 @@ export const createImportWalletCommand: (
   'Import (restore) wallet from mnemonic phrase',
   [
     globalOptions.keyMnemonic(),
-    globalOptions.passwordFileRepeat({ isOptional: false }),
+    securityOptions.createPasswordOption({
+      message: 'Enter the new wallet password',
+      confirmPasswordMessage: 'Enter the password again',
```
Comment (realdreamer) [inline packages/tools/kadena-cli/src/wallets/commands/walletsImportWallet.ts:75]:
```suggestion
      confirmPasswordMessage: 'Re-enter the password again',
```

File: packages/tools/kadena-cli/src/wallets/commands/walletsDecrypt.ts [hunk 3]
```diff
@@ -41,7 +41,9 @@ export const createDecryptCommand: (program: Command, version: string) => void =
     'Decrypt message',
     [
       globalOptions.message({ isOptional: false }),
-      globalOptions.currentPasswordFile({ isOptional: false }),
+      securityOptions.createPasswordOption({
+        message: 'Enter the password used to encrypt',
```
[the remark this answers is not in the document]
Comment (barthuijgen) [reply packages/tools/kadena-cli/src/wallets/commands/walletsDecrypt.ts:45]: `kadena wallet decrypt` actually has nothing to do with wallet, it just takes an arbitrary encrypted string

**Review by realdreamer — APPROVED** [review]

File: packages/tools/kadena-cli/src/wallets/commands/walletsChangeWalletPassword.ts [hunk 4]
```diff
@@ -80,8 +80,13 @@ export const createChangeWalletPasswordCommand: (
...
   [
     globalOptions.walletSelect(),
-    globalOptions.currentPasswordFile({ isOptional: false }),
-    globalOptions.newPasswordFile({ isOptional: false }),
+    securityOptions.createPasswordOption({
+      message: 'Enter your current password',
+    }),
+    securityOptions.createNewPasswordOption({
+      message: 'Enter the new wallet password',
+      confirmPasswordMessage: 'Re-enter the new password',
```
Comment (realdreamer) [inline packages/tools/kadena-cli/src/wallets/commands/walletsChangeWalletPassword.ts:88]:
```suggestion
      confirmPasswordMessage: 'Re-enter the password',
```

**Review by realdreamer — APPROVED** [review]

**Pull request merged**